src/bitcoin-cli.cpp

```diff
@@ -648,6 +666,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "                                  2 - Like 1 but with an address column\n"
         "                                  3 - Like 1 but with a version column\n"
         "                                  4 - Like 1 but with both address and version columns\n"
+        "                                  5 - Like 4 but limited to outbound peers only, to save screen space\n"
```

Comment:
Before these numbers somehow designated increasing verbosity and more columns. Whereas now the newly added "5" does not add more columns, but _removes rows_ compared to "4".

I am not sure how to improve that. It is a different dimension, maybe use a different option "outonly". Then one could have either one of the 0-4 modes in "outonly" mode or in full mode (in + out).

(non-blocker, just something to consider)

Reply:
I see, but think it's simpler and more convenient for the user (i.e. me) to type 5 rather than 4, or vice versa.

I did maintain the "high details levels = maximum" behavior preferred by @laanwj in https://github.com/bitcoin/bitcoin/pull/21192 for values above 5.

Reply:
Also, I find myself only using level 4 or 5. Perhaps other reviewers would prefer the option to apply, as you suggest, to all the other levels. Happy to do that (here or in a follow-up) if people prefer.

Reply:
Updated to take @vasild's proposal. I believe it's the best path of the 3 options discussed above.

Reply:
> I did maintain the "high details levels = maximum" behavior preferred by laanwj in https://github.com/bitcoin/bitcoin/pull/21192 for values above 5.

Thank you 😄